MIP113/MIP113.md

```diff
@@ -186,28 +187,46 @@ Content: CVC Creation
+	* **4.7.3.2**: CVC Members may vote by posting a signed message referring the proposal and containing either "For" or "Against".
 ---
-* **4.5.3:** The available CVC Decisions are detailed in the subelements of this clause.
-	* **4.5.3.1:** *Adding a CVC Member*. A simple majority of existing CVC Members can onboard a new CVC Member.
-	* **4.5.3.2:** *Removing CVC Member*. A simple majority of existing CVC Members can remove a CVC Member.
-	* **4.5.3.3:** *Communication infrastructure moderation rights management*. A simple majority of CVC Members can add or remove one or more moderators for one or more platforms.
-	* **4.5.3.4:** *Scheduling Subcommittee Meeting*. A simple majority of CVC Members can schedule the time of a Subcommittee Meeting.
-	* **4.5.3.5:** *Ratifying Position Paper*. A simple majority of CVC Members can ratify a Position Paper that has been publicly posted to the Maker Governance Forum.
-	* **4.5.3.6:** *CD rejection*. A simple majority of CVC Members can reject a CD.
-	* **4.5.5.7:** *CVC Split*. 25% of the CVC Member Voting power can initiate a CVC Split. A second vote then occurs where all CVC Members can choose to join the split. If the split gains more than 50% of all voting power, it will take over the name and CD followers.
-
-##### 4.6: CVC Member participation rewards
-
-The 14 CVC Members that have the highest amounts of verified MKR, meet minimum participation requirements and have not opted out, are eligible to receive CVC Member participation rewards.
-* **4.6.1:** A total of 500 MKR per year is available per year, distributed equally across 14 slots for each Scope per quarter. This gives a reward of 0.725 MKR available per qualifying CVC Member, per scope, per quarter.
-* **4.6.2:** The minimum participation requirement for a Scope each quarter, is to participate in a scheduled quarterly Subcommittee Call for the specific Scope, and to have co-authorship of the Scope Framework Position Document for of the specific Scope.
-* **4.6.3:** The Arbitration Facilitators must monitor the participation of CVC Members and bundle the MKR payments of the CVC Members that qualify into Executive Votes each quarter.
+* **4.7.3:** The available CVC Decisions are detailed in the subelements of this clause.
+	* **4.7.3.1:** *Adding a CVC Member*. A simple majority of existing CVC Members can onboard a new CVC Member.
+	* **4.7.3.2:** *Removing CVC Member*. A simple majority of existing CVC Members can remove a CVC Member.
+	* **4.7.3.3:** *Communication infrastructure moderation rights management*. A simple majority of CVC Members can add or remove one or more moderators for one or more platforms.
+	* **4.7.3.4:** *Scheduling Subcommittee Meeting*. A simple majority of CVC Members can schedule the time of a Subcommittee Meeting.
+	* **4.7.3.5:** *Ratifying Position Paper*. A simple majority of CVC Members can ratify a Position Paper that has been publicly posted to the Maker Governance Forum.
```

Comment:
Should the strategy require a stronger signal from the CVC? Thinking of fringes where a CVC is starting to split but it hasn't yet, assuming there are a few "swing voters" it wouldn't be too hard to pass two significantly different strategies from the CVC in a short period of time. Not sure it's too much of a problem but could lead to extra arbitration

Reply:
These are just copied from previous version with structural fixes.